flink-metrics/flink-metrics-datadog/pom.xml

```diff
@@ -88,6 +65,7 @@ under the License.
 							<goal>shade</goal>
 						</goals>
 						<configuration>
+							<promoteTransitiveDependencies>true</promoteTransitiveDependencies>
```

Comment:
Why is this needed now?

Reply:
This is a cleaner approach compared to listing transitive dependencies. We never had a dependency on `okio`; it was only defined to be included in the shaded jar. Promoting transitive dependencies accomplishes the same thing, without the maintainability issues that arise when you bump the version of a dependency.

Reply:
sounds good!